src/Text/Pandoc/Writers/Custom.hs

```diff
@@ -232,7 +232,9 @@ inlineToCustom (Link attr txt (src,tit)) =
 inlineToCustom (Image attr alt (src,tit)) =
   callFunc "Image" (Stringify alt) src tit (attrToMap attr)
 
-inlineToCustom (Note contents) = callFunc "Note" (Stringify contents)
+inlineToCustom (Note FootNote contents) = callFunc "FootNote" (Stringify contents)
```

Comment:
This approach will break existing custom writers that use Note.
A better approach would be `callFunc "Note" "footnote"` and `callFunc "Note" "endnote"`, I think.
This would allow current writers to work without being changed.  @tarleb, any thoughts?

Reply:
That seems like a nice solution.  We'd get full backwards compatibility if we pass the note type after the content:

```haskell
inlineToCustom (Note noteType contents) = callFunc "Note" (Stringify contents) noteType
```

Lua won't mind the extra parameter if the function just expects contents. This would pass the node type as "FootNote" and "EndNote*.

Reply:
Fixed